src/arch/riscv/pcstate.hh

```diff
@@ -75,7 +75,7 @@ class PCState : public GenericISA::UPCState<4>
         _vtype(other._vtype), _vl(other._vl)
     {}
     PCState &operator=(const PCState &other) = default;
-    PCState() = default;
+    PCState() { _rvType = RV64; };
```

Comment:
I don't understand the need for this change. On line 67, we set `_rvType` to 64 by default. Could you explain why this line is needed?